src/protocol.cpp

```diff
@@ -46,46 +46,44 @@ const char *CFCHECKPT="cfcheckpt";
 const char *WTXIDRELAY="wtxidrelay";
 } // namespace NetMsgType
 
-/** All known message types. Keep this in the same order as the list of
+/** All known message types including the short-ID. Keep this in the same order as the list of
```

Comment:
Please add a reference in a doc-comment that these are defined in BIP324, section Message-Type-ID.

Reply:
Done.